board/apiwrappers.py

```diff
@@ -0,0 +1,115 @@
+from requests import get, post
+
+
+API_IGDB_URL = 'https://api.igdb.com/v4/'
+API_TWITTER_URL = 'https://api.twitter.com/2/'
+
+
+class BaseWrapper:
+    def __init__(self, auth_token):
+        self.auth_token = auth_token
+        self._requests_method = get
```

Comment:
Some apis may contain endpoints with different methods. I think you can just pass this method as a parameter to `_api_request` method and also create `_get` and `_post` methods for convenience in this class.